verification/verify.go

```diff
@@ -29,7 +29,11 @@ type Retryable struct {
 	error
 }
 
-var ErrPixelMismatch = Retryable{errors.New("PixelMismatch")}
+type Fatal struct {
```

Comment:
I'm not too familiar with the codebase here but couldn't we encapsulate both these types in a single type just like we do for acceptable/unacceptable payment errors? 

I also find the language use to be sub-optimal. I don't think a fatal error should ever be retryable, such is the definition of fatal. 

I wish I had something more constructive here to tell you or an alternative but I come with empty hands for now.

Reply:
Should we call it `FatalRetryable` instead of `Fatal` ?

Reply:
As discussed during planning, I'll leave it up to you ! :)